src/plugins/intel_gpu/src/graph/scatter_elements_update.cpp

```diff
@@ -34,6 +36,32 @@ layout scatter_elements_update_inst::calc_output_layout(scatter_elements_update_
     return layout{output_type, input_format, output_shape};
 }
 
+template<typename ShapeType>
+std::vector<layout> scatter_elements_update_inst::calc_output_layouts(scatter_elements_update_node const& /*node*/, const kernel_impl_params& impl_param) {
+    auto input0_layout = impl_param.get_input_layout(0);
+    auto input1_layout = impl_param.get_input_layout(1);
+    auto input2_layout = impl_param.get_input_layout(2);
+    // axis should be scalar or 1D tensor
+    auto input3_layout = layout{ov::PartialShape{1}, data_types::i64, format::bfyx};
+
+    std::vector<ShapeType> input_shapes = {
+        input0_layout.get<ShapeType>(),     // inputs_shape
+        input1_layout.get<ShapeType>(),     // indices_shape,
+        input2_layout.get<ShapeType>(),     // updates_shape,
+        input3_layout.get<ShapeType>(),     // axis_shape,
+    };
+
+    std::vector<ShapeType> output_shapes = {ShapeType()};
+
+    ov::op::v3::ScatterElementsUpdate op;
+    output_shapes = shape_infer(&op, input_shapes);
+
+    return { layout{output_shapes[0], input0_layout.data_type, input0_layout.format} };
```

Comment:
I think scatter update 's output shape is always same as original input, isn't it?

Reply:
So I think we just can do it as static shape just replacing the access to the "tensor()" to get_partial_shape().